tests/ctags/runner.sh

```diff
@@ -18,6 +18,9 @@ mkdir -p "$CONFDIR/filedefs/" || exit 99
 # are no extension patterns, like e.g. Meson.
 sed 's/^\([^=[]\{1,\}\)\(=[^;]\{1,\}\(;[^;]\{1,\}\)*\);*$/\1\2;*.\1_unittest;/' \
   < "${top_srcdir:-../..}"/data/filetype_extensions.conf > "$CONFDIR/filetype_extensions.conf" || exit 99
+# add HCL filetype extension for the Terraform parser test (HCL is not an official Geany filetype)
+echo 'HCL=*.tf;*.tfvars;*.hcl;' >> "$CONFDIR/filetype_extensions.conf"
```

Comment:
I feel dirty :(.

Though I had to add this to make Geany recognize the file extension and map to the correct parser.

Reply:
As @techee said, why not add the filetype? Is it of too poor quality and cannot be made better, or is it a question of how much the language is spread?
If it's useful I don't see why not adding it ptoperly rather than adding a hack like that :)

Reply:
> As @techee said, why not add the filetype? Is it of too poor quality and cannot be made better, or is it a question of how much the language is spread? If it's useful I don't see why not adding it ptoperly rather than adding a hack like that :)

It was a question of "how much the language is spread" for me.
But if we agree that it's fine to add a custom filetype, I'd be happy.